draft-kalos-bbs-blind-signatures.md

```diff
@@ -440,22 +436,19 @@ Deserialization:
 
 Procedure:
 
-1.  message_scalars = BBS.messages_to_scalars(messages, api_id)
-
-2.  committed_message_scalars = ()
-3.  committed_message_scalars.append(secret_prover_blind)
-4.  committed_message_scalars.append(BBS.messages_to_scalars(
-                                            committed_messages, api_id))
-
-
-5.  generators = BBS.create_generators(length(message_scalars) + 1, api_id)
-6.  blind_generators = BBS.create_generators(length(committed_message_scalars) + 1, "BLIND_" || api_id)
+1. (message_scalars, generators) = prepare_parameters(
+                                         messages,
+                                         committed_messages,
+                                         length(messages) + 1,
+                                         length(committed_messages) + 1,
+                                         secret_prover_blind,
+                                         api_id)
 
-7.  indexes = ()
-8.  indexes.append(disclosed_indexes)
-9.  for j in disclosed_commitment_indexes: indexes.append(j + L + 1)
+2. indexes = ()
+3. indexes.append(disclosed_indexes)
+4. for j in disclosed_commitment_indexes: indexes.append(j + L + 1)
 
-10. proof = BBS.CoreProofGen(
+5. proof = BBS.CoreProofGen(
```

Comment:
Don't we need to update the parameters to BBS.CoreProofGen to use the message_scalars and generators from prepare_parameters?

Reply:
Yeap, fixed! Thank you for catching this!